docs/pages/reference/cli.mdx

```diff
@@ -1,53 +1,68 @@
+| `teleport version` | Prints the current release version of the Teleport binary installed on your system. |
+| `teleport app start` | Starts the application proxy service. |
+| `teleport db start` | Starts the Teleport Database Service. |
+| `teleport db configure create` | Generates a configuration YAML file for the Database Service. This file should be customized in production to suit the needs of your environment, and the default output should only be used when testing. |
+| `teleport db configure bootstrap` | Used to bootstrap a configuration to the Teleport Database Service by reading a provided configuration. |
+| `teleport db configure aws print-iam` | Generates and outputs current IAM policies for a Teleport-managed database. |
+| `teleport db configure aws create-iam` | Generates, creates, and attaches desired IAM policies to a Teleport-managed database. |
+| `teleport install systemd` | Creates a systemd unit file, used to configure and install a `teleport` service daemon. |
+| `teleport node configure` | Generates a configuration YAML file for a Teleport Node accessed via SSH. This file should be customized in production to suit the needs of your environment, and the default output should only be used when testing. |
+
+<Notice type="tip">
+For more information on subcommands when working with the `teleport` cli, use the `--help` option or `teleport <subcommand> --help`.
 </Notice>
 
-| Service | Role Name | Description |
-| - | - | - |
-| [Node](../architecture/nodes.mdx) | `node` | Runs a daemon on a host that allows SSH connections from authenticated clients. |
-| [Auth](../architecture/authentication.mdx) | `auth` | Authenticates hosts and users who want access to Teleport-managed resources or information about a cluster. |
-| [Proxy](../architecture/proxy.mdx) | `proxy` | The gateway that clients use to connect to the Auth Service or resources managed by Teleport. |
-| [App](../application-access/introduction.mdx) | `app` | Runs a daemon on a host that provides access to applications using an SSH reverse tunnel. |
-| [Kube](../kubernetes-access/introduction.mdx) | `kube` | The Teleport daemon will run the Kubernetes Service. |
-| [DB](../database-access/reference.mdx) | `db` | The Teleport daemon will run the Database Service. |
-| [Trusted Cluster](../management/admin/trustedclusters.mdx) | `trusted_cluster` | The Teleport daemon will support a leaf cluster used to connect to another Teleport cluster. |
-| [Windows Desktop](../desktop-access/getting-started.mdx) | `windowsdesktop` | The Teleport daemon will run the Windows Desktop Service. |
-
 ### teleport start
 
+The `teleport start` command includes a large number of optional configuration flags which change the behavior of the default command to support a wider spectrum of Teleport services.
```

Comment:
```suggestion
The `teleport start` command includes a large number of optional configuration flags.
```
I think we can make this sentence much more concise. Do we need more than this part of it? For me, the crux of the paragraph is that `teleport start` is unwieldy and using a configuration file is preferred.